loadrunner_professional/README.md

```diff
@@ -0,0 +1,57 @@
+# LoadRunner Professional
+
+## Overview
+
+OpenText™ LoadRunner Professional is a load testing solution that enables you to test the performance of diverse application types, and identify and resolve issues before applications go live.
+
+LoadRunner Controller is a tool for creating and controlling LoadRunner Professional scenarios. A scenario defines the events that occur during each testing session. It controls the number of users to emulate (virtual users, or Vusers), the actions they perform, and the machines on which they run their emulations. You use scenarios to create load tests to check the reliability and strength of your servers.
+
+With this integration, Controller pushes real-time metrics and data from scenario runs to Datadog.
+
+|   |   |
+|---|---|
+|__Send scenario information__| Send information about the scenario run, such as the start and stop time, duration, and included scripts. |
```

Comment:
Is the "information" described here sent in the form of logs? If so, i'd mention that explicitly. If not, would be helpful to mention in the Overview what the value of the submitted logs are